src/main/kotlin/com/autonomousapps/internal/advice/ConsoleReport.kt

```diff
@@ -0,0 +1,70 @@
+package com.autonomousapps.internal.advice
```

Comment:
wrong package. I meant either
1. `com.autonomousapps.advice` for something that is public-facing; or
2. `com.autonomousapps.internal.models.kt` if it's only for internal use.

Note that the second one refers to a _file_ named `models.kt`, which has a bunch of `data class` es in it.

Reply:
I moved it to the `com.autonomousapps.internal.models.kt` file. I really hesitated, just because the data is serialized but at the same time it doesn't make much sense to consume this report as it can be regenerated from the advice data directly.